src/ucp/tag/tag_send.c

```diff
@@ -340,11 +288,13 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_sync_nb,
 
     ucp_tag_send_req_init(req, ep, buffer, datatype, count, tag,
                           UCP_REQUEST_FLAG_SYNC);
+    req->user_data = NULL;
```

Comment:
set user data as in tag_send (use common inline/macro)?

Reply:
> set user data as in tag_send (use common inline/macro)?

ok, initialisation is moved into ```ucp_tag_send_req_init```